src/pages/i18n/Drawer.tsx

```diff
@@ -73,6 +83,11 @@ export const Drawer = (props: DrawerProps): JSX.Element => {
                         <Typography variant={'subtitle1'} color={'inherit'}>
                             {t('I18N')}
                         </Typography>
+                        <div>
```

Comment:
Perhaps we can use a Typography element

Reply:
Good idea, I've updated to use the subtitle1 variant.